.gitignore

```diff
@@ -31,3 +31,7 @@ bwapi/libReplayToolTest/TestData/Replay_Tool_Actual.exe
 bwapi/ipch/
 Release_Binary/Chaoslauncher/__log.txt
 Release_Binary/ExampleAIClient/Source/
+
+/bwapi/packages/*
+!/bwapi/packages/repositories.config

```

Comment:
I've removed the directory in 4733987f27ab168312ad54e61a8370ec7615a60d. You will have to rebase your changes.